go.mod

```diff
@@ -14,7 +14,7 @@ require (
 
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/CycloneDX/cyclonedx-go v0.6.0 // indirect
+	github.com/CycloneDX/cyclonedx-go v0.7.0 // indirect
```

Comment:
@joshuatcasey I'm surprised we don't have to pin this to `v0.6.0` like we did everywhere else. I accept that it seems to work, but i was wondering if you had thoughts on this?

Reply:
There's a `replace` directive at the bottom of `go.mod`. Rather confusingly, `go mod` updates the `require` directive with the latest version, but the `go.sum` seems to keep the `replaced` version.

Reply:
Ah, yes, i missed because it wasn't shown in the diff by default (it didn't change) and it was at the bottom whereas in other `go.mod` files we've explicitly put it at the top.

I'll approve this PR.